spec/oas_v1.yaml

```diff
@@ -0,0 +1,497 @@
+openapi: 3.1.0
+info:
+  title: Agora API
+  description: Public API for interacting with the Agora platform
+  version: 0.1.0-beta
+  license:
+    name: MIT
+    url: https://opensource.org/licenses/MIT
+servers:
+  - url: vote.optimisim.io/api/v1
+    description: Base URL for optimism production
+security:
+  - ApiKeyAuth: []
+components:
+  securitySchemes:
+    ApiKeyAuth:
+      type: apiKey
+      in: header
+      name: Authorization
+  schemas:
+    PageMetadata:
+      summary: Pagination metadata
+      description: Metadata associated with paginated requests.
+      type: object
+      properties:
+        hasNext:
+          summary: Flag indicating if there's more data for retieval
+          description: A boolean flag indicating if there is additional data past the returned page to retrieve.
+          type: boolean
+        totalReturned:
+          summary: Total records returned
+          description: A number indicating the total amount of records returned for the request.
+          type: integer
+          format: int32
+        nextOffset: 
+          summary: Offset to supply to the next request
+          description: A number indicating the offset at which a subsequent request may retrieve the next set of records.
+          type: integer
+          format: int32
+    Delegation:
+      summary: A delegation of voting power
+      description: A delegation of voting power from one address to another.
+      type: object
+      properties:
+        address: 
+          summary: Delegate address
+          description: Address of the delegator or delegatee.
+          type: string 
+          examples:
+          - summary: Address of the delegator or delegatee.
+            value: "0xDa6d1F091B672C0f9e215eB9fa6B5a84bF2c5e11"
+        totalAllowance:
+          summary: Voting allowance for delegate
+          description: Total amount of voting power delegated to or from address. 
+          type: string
+          examples:
+          - summary: Amount of voting power delegated to or from address.
+            value: "100000"
+        delegatedOn:
+          summary: When delegation occured
+          description: The datetime on which the delegation of voting power occurred. 
+          type: string
+          format: date-time
+          examples:
+          - summary: The datetime on which the delegation of voting power occurred.
+        type:
+          summary: Type of delegation
+          description: The type of delegation; can be advanced or standard.
+          type: string
+          enum:
+            - advanced
+            - standard
+        amount:
+          summary: Amount delegated
+          description: The amount of voting power delegated; can be full or partial.
+          type: string
+          enum:
+            - full
+            - partial
+    BaseDelegate:
```

Comment:
That was the initial intent, in case there are things that we don't expand in the paginated GET /delegates which we do want to expand in the scoped /delegates/{address} request. will re-review if this is necessary